cunumeric/array.py

```diff
@@ -1228,25 +1267,7 @@ def nonzero(self) -> tuple[ndarray, ...]:
             ndarray(shape=thunk.shape, thunk=thunk) for thunk in thunks
         )
 
-    def __nonzero__(self):
```

Comment:
This was removed because it does not exist on `numpy.ndarray` (not sure why it is here)